hudi-client/hudi-java-client/src/test/java/org/apache/hudi/client/TestJavaHoodieBackedMetadata.java

```diff
@@ -1280,6 +1282,7 @@ public void testFailedBootstrap() throws Exception {
       List<HoodieRecord> records = dataGen.generateInserts(newCommitTime, 100);
       client.startCommitWithTime(newCommitTime);
       List<WriteStatus> writeStatuses = client.insert(records, newCommitTime);
+      client.commit(newCommitTime, writeStatuses);
```

Comment:
why insert/bulk_insert returns list of write status instead of `HoodiWriteData`.

Reply:
That is inline with the API signature of insert and bulk insert